includes/class-wp-security-bp-users.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * The file that defines the Users class
+ *
+ * This class handles all the users requests.
+ *
+ * @link       http://example.com
+ * @since      1.0.0
+ *
+ * @package    wp_security_bp
+ * @subpackage wp_security_bp/includes
+ */
+
+/**
+ * The Users class.
+ *
+ * This class handles all the users requests.
+ *
+ * @since      1.0.0
+ * @package    wp_security_bp
+ * @subpackage wp_security_bp/includes
+ * @author     Cyan Lovers <hello@cyanlove.com>
+ */
+class WP_Security_BP_Users {
+	/**
+	 * The unique identifier of this plugin.
+	 *
+	 * @since    1.0.0
+	 * @access   protected
+	 * @var      string    $plugin_name    The string used to uniquely identify this plugin.
+	 */
+	protected $plugin_name;
+
+	/**
+	 * Short desc
+	 *
+	 * @since    1.0.0
+	 * @access   protected
+	 * @var      string    $request_uri    The URI where the request is made from
+	 */
+	protected $request_uri;
+
+	/**
+	 * Short desc
+	 *
+	 * @since    1.0.0
+	 * @access   protected
+	 * @var      string    $nonce_action_name    The action for the nonce creation
+	 */
+	protected $nonce_action_name;
+
+	/**
+	 * Short desc
+	 *
+	 * @since    1.0.0
+	 * @access   protected
+	 * @var      string    $users    The array of all users info
+	 */
+	protected $users;
+
+	/**
+	 * Short desc
+	 *
+	 * @since    1.0.0
+	 * @access   protected
+	 * @var      integer    $range_scan_id  The integer to set the range to scan of users id's.
+	 */
+	protected $range_scan_id = 5;

```

Comment:
Attributes should be assigned in constructor? 🤔

Reply:
I didn't agree this change. this may be a static attr... and assigned at instantiating moment... don't you think?...

anyway, to late...